src/chains.ts

```diff
@@ -1,4 +1,8 @@
-import { ChainId } from "./constants";
+enum ChainId {
```

Comment:
`ChainId` is also defined in `src/constants.ts`. we should only keep it at one place